games-fps/freedm/freedm-0.11.3-r1.ebuild

```diff
@@ -13,7 +13,7 @@ KEYWORDS="~amd64 ~x86"
 RDEPEND="
 	~games-fps/freedm-data-${PV}
 	|| (
-		games-fps/gzdoom
+		games-fps/gzdoom[nonfree]
```

Comment:
You can avoid forcing the gzdoom update by changing this to `games-fps/gzdoom[nonfree(+)]`.